outputs.tf

```diff
@@ -12,3 +12,8 @@ output "task_role_arn" {
   description = "ECS Task role ARN"
   value       = "${aws_iam_role.ecs_task.arn}"
 }
+
+output "ecs_sg_id" {
```

Comment:
Rename to service_security_group_id. Generally we avoid abbreviations unless we are just mirroring the original variable or output. Also, since none of the other outputs have the ecs prefix, I think it’s more consistent if we drop it here.